### PR TITLE
New package: pmw-5.20

### PR DESCRIPTION
**Pull request by spaskalev**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, amd64-glibc

### CODE REVIEW

File: srcpkgs/pmw/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+revision=1
+build_style=gnu-configure
+make_install_args="MANDIR=/usr/share/man"
+short_desc="PMW is a computer program for high quality music typesetting"
+maintainer="Stanislav Paskalev <spaskalev@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="http://quercite.dx.am/pmw.html"
+distfiles="https://github.com/PhilipHazel/pmw/releases/download/pmw-${version}/pmw-${version}.tar.gz"
+checksum=a78063bcd59b72ad31e6ad471270bfc0d28db38ecb74cec20e06503ad033c8b4
+post_install() {
```
Comment (paper42) [inline srcpkgs/pmw/template:13]:
```suggestion

post_install() {
```

File: srcpkgs/pmw/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'pmw'
+pkgname=pmw
+version=5.20
+revision=1
+build_style=gnu-configure
+make_install_args="MANDIR=/usr/share/man"
```
Comment (paper42) [inline srcpkgs/pmw/template:6]: it doesn't seem to put any manpages to the package, is it putting it to / directly and not into $DESTDIR?

File: srcpkgs/pmw/template (hunk 1 above)
Comment (spaskalev) [reply srcpkgs/pmw/template:14]: Fixed

File: srcpkgs/pmw/template (hunk 2 above)
Comment (spaskalev) [reply srcpkgs/pmw/template]: It should be okay now

File: srcpkgs/pmw/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'pmw'
+pkgname=pmw
+version=5.20
+revision=1
+build_style=gnu-configure
+short_desc="PMW is a computer program for high quality music typesetting"
```
Comment (classabbyamp) [inline srcpkgs/pmw/template:6]:
```suggestion
short_desc="program for high quality music typesetting"
```

File: srcpkgs/pmw/template [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'pmw'
+pkgname=pmw
+version=5.20
+revision=1
+build_style=gnu-configure
```
Comment (classabbyamp) [inline srcpkgs/pmw/template:5]:
```suggestion
build_style=gnu-configure
make_install_args="MANDIR=\${DESTDIR}/usr/share/man"
```

File: srcpkgs/pmw/template [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+short_desc="PMW is a computer program for high quality music typesetting"
+maintainer="Stanislav Paskalev <spaskalev@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="http://quercite.dx.am/pmw.html"
+distfiles="https://github.com/PhilipHazel/pmw/releases/download/pmw-${version}/pmw-${version}.tar.gz"
+checksum=a78063bcd59b72ad31e6ad471270bfc0d28db38ecb74cec20e06503ad033c8b4
+
+pre_install() {
+	make_install_args="MANDIR=${DESTDIR}/usr/share/man"
+}
```
Comment (classabbyamp) [inline srcpkgs/pmw/template:15]:
```suggestion
```

File: srcpkgs/pmw/template [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'pmw'
+pkgname=pmw
+version=5.20
+revision=1
+build_style=gnu-configure
+short_desc="PMW is a computer program for high quality music typesetting"
+maintainer="Stanislav Paskalev <spaskalev@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="http://quercite.dx.am/pmw.html"
```
Comment (classabbyamp) [inline srcpkgs/pmw/template:9]:
```suggestion
homepage="http://quercite.dx.am/pmw.html"
changelog="https://raw.githubusercontent.com/PhilipHazel/pmw/main/doc/ChangeLog"
```

Comment (spaskalev) [reply srcpkgs/pmw/template:10]: Done

File: srcpkgs/pmw/template (hunk 5 above)
Comment (spaskalev) [reply srcpkgs/pmw/template]: Done

File: srcpkgs/pmw/template (hunk 4 above)
Comment (spaskalev) [reply srcpkgs/pmw/template:5]: Thanks, done

File: srcpkgs/pmw/template (hunk 3 above)
Comment (spaskalev) [reply srcpkgs/pmw/template]: Fixed, with an upper-case first letter (xlint fails otherwise)

**Comment by spaskalev** [comment]
ping :)

**Comment by spaskalev** [comment]
ping :)

File: srcpkgs/pmw/template [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+short_desc="Program for high quality music typesetting"
+maintainer="Stanislav Paskalev <spaskalev@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="http://quercite.dx.am/pmw.html"
+changelog="https://raw.githubusercontent.com/PhilipHazel/pmw/main/doc/ChangeLog"
+distfiles="https://github.com/PhilipHazel/pmw/releases/download/pmw-${version}/pmw-${version}.tar.gz"
+checksum=a78063bcd59b72ad31e6ad471270bfc0d28db38ecb74cec20e06503ad033c8b4
+
+post_install() {
+	vinstall doc/spec.pdf 644 /usr/share/pmw manual.pdf
```
Comment (classabbyamp) [inline srcpkgs/pmw/template:16]: use vdoc, unless this is where the application looks for this (if it does so)

Comment (spaskalev) [reply srcpkgs/pmw/template]: Done (this is a user manual, the application does not depend on it)

**Pull request merged**